editor/src/core/shared/element-template.ts

```diff
@@ -617,6 +617,10 @@ export function isJSXAttributeValue(element: JSXElementChild): element is JSExpr
   return element.type === 'ATTRIBUTE_VALUE'
 }
 
+export function isNullJSXAttributeValue(element: JSXElementChild): boolean {
```

Comment:
This function is what I was thinking of, but it should probably use `jsxSimpleAttributeToValue` to cater for more cases.

Reply:
would this work? https://github.com/concrete-utopia/utopia/pull/3512/commits/1504c93a00ae02a09442cf1bfc831db2fada9872